src/main/java/com/algomart/kibouregistry/services/implementations/ParticipantsServiceImpl.java

```diff
@@ -0,0 +1,65 @@
+package com.algomart.kibouregistry.services.implementations;
+
+import com.algomart.kibouregistry.dao.ParticipantsRepo;
+import com.algomart.kibouregistry.entity.Participants;
+import com.algomart.kibouregistry.exceptions.ResourceNotFoundException;
+import com.algomart.kibouregistry.services.ParticipantsService;
+import jakarta.transaction.Transactional;
+import lombok.AllArgsConstructor;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+@AllArgsConstructor
+public class ParticipantsServiceImpl implements ParticipantsService {
+
+    private final ParticipantsRepo participantsRepo;
+
+    @Override
+    @Transactional
+    @PreAuthorize("hasRole('ADMIN')")
```

Comment:
Why are you adding @PreAuthorize at the service layer? Remove it from everywhere you used it.